tests/foreman/api/test_ansible.py

```diff
@@ -86,6 +86,8 @@ def test_positive_ansible_job_on_host(target_sat, module_org, rhel_contenthost):
         2. Job execution must be successful.
 
     :CaseAutomation: Automated
+
+    :CaseImportance: Critical
```

Comment:
Can you also change the case importance to critical for CLI

Reply:
It's there already :)

Reply:
I see High right here? https://github.com/SatelliteQE/robottelo/pull/10562/files#diff-272a921164230d68c640329237c2fff2f535796cd0786a9718455d17abd2a2e1R15

Reply:
Is that not UI?

Reply:
That link is broken sorry. Try this one: https://github.com/SatelliteQE/robottelo/blob/6666afc8f335a9b2e5aca1bf988f64d9ad09f57d/tests/foreman/cli/test_ansible.py#L15

Reply:
Oh ok. I thought you were changing to critical because of the component tier or something. Makes sense.